src/backend/base/langflow/components/embeddings/ollama.py

```diff
@@ -13,16 +20,19 @@ class OllamaEmbeddingsComponent(LCModelComponent):
     name = "OllamaEmbeddings"
 
     inputs = [
-        MessageTextInput(
-            name="model",
+        DropdownInput(
+            name="model_name",
             display_name="Ollama Model",
-            value="nomic-embed-text",
+            value="",
+            options=[],
+            real_time_refresh=True,
+            refresh_button=True,
```

Comment:
This will help people use custom models
```suggestion
            value="",
            options=[],
            real_time_refresh=True,
            refresh_button=True,
            combobox=True,
```